README.md

```diff
@@ -1,655 +1,288 @@
 # gage conditions
 
+This repo contains code used to create the recurring `U.S. River Conditions` animation series, an example of which can be seen [here](https://www.usgs.gov/media/videos/us-river-conditions-january-march-2022). The name of this repo is a bit of a misnomer because at present this code will produce a video file, not a gif. However, the code used to create a gif version is still available in `6_visualize/src/combine_animation_frames.R` > `combine_animation_frames_gif()`.
 
-The master repo is setup to build a video file. I (Lindsay) have been running the following lines of code to build the big pieces individually. One version of the final product can be found [here](www.usgs.gov/media/videos/us-river-conditions-water-year-2018). Before running this, you need delete the old contents of `6_visualize/tmp`. 
+*Instructions generated by Lindsay Platt*
 
-```r
-#####################
-## Download data
+## Before you build
 
-# Uses GD still so need to do this first one separately so it can log in
-scipiper::scmake("1_fetch/out/dv_data.rds.ind", "1_fetch.yml")
+Note that before you can just build this as the code suggests, you will have to have the appropriate permissions to our intermediate data products. Mainly, this is access to the Google Drive folder which we grant to internal users and close collaborators.
 
-# Then all the rest
-scipiper::scmake("1_fetch/out/dv_data_fixed_gh.rds.ind", "1_fetch.yml")
-scipiper::scmake('1_fetch/out/sites_stage.rds.ind', remake_file = '1_fetch.yml')
-scipiper::scmake("2_process/out/dv_stats.rds.ind", "2_process.yml")
-scipiper::scmake("2_process/out/dv_stats_fixed_gh.rds.ind", "2_process.yml")
-scipiper::scmake("2_process/out/dv_stat_styles.rds.ind", "2_process.yml")
+## How to build this animation
 
-#####################
-## Create a callouts file (it can be empty if there are none)
+The process to create this animation is almost entirely automated using the (now dormant) custom dependency management R package, `scipiper`. There is a bit of manual work required to create and then prepare the animation's event/text callouts. Otherwise, it is mostly running chunks of code. The entire process is outlined below (*steps that aren't code, but are human checks are italicized in parentheses*).
```

Comment:
```suggestion
The process to create this animation is almost entirely automated using the (now dormant) custom dependency management R package, [`scipiper`](https://github.com/USGS-R/scipiper). There is a bit of manual work required to create and then prepare the animation's event/text callouts. Otherwise, it is mostly running chunks of code. The entire process is outlined below (*steps that aren't code, but are human checks are italicized in parentheses*).
```